compiler/psi/src/org/jetbrains/kotlin/psi/KtParameter.java

```diff
@@ -144,6 +144,28 @@ public boolean isCatchParameter() {
         return getParent().getParent() instanceof KtCatchClause;
     }
 
+    /**
+     * For example,
+     *   lambdaConsumer { lambdaParameter ->
+     *     ...
+     *   }
+     *
+     * @return [true] if this [KtParameter] is a parameter of a lambda.
+     */
+    public boolean isLambdaParameter() {
+        return getParent().getParent() instanceof KtFunctionLiteral;
```

Comment:
Aha, thanks. Then, since this pattern came from the above util, I think that one should be fixed together. Actually, all these three share the same logic, so I rewrote those with a common util (where `null` from `getParent()` is checked).